src/main/scala/com/atomist/rug/spi/Handlers.scala

```diff
@@ -34,6 +49,11 @@ object Handlers {
 
   sealed trait Instruction {
     def detail: Detail
+
+    def toDisplay: String = {
+      val name = this.getClass.getName.split('$').last
```

Comment:
Those have been pretty stable so far. If the name did change then I would probably want to display what it was changed to. Edit, Generate...